torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -36,25 +37,24 @@ namespace jit {
 
 char const* toString(OpCode op);
 
-namespace {
-
-ExportModuleExtraFilesHook& GetExtraFilesHook() {
-  static ExportModuleExtraFilesHook func = nullptr;
-  return func;
-}
-
-static IValue Tup(std::vector<IValue> ivalues) {
```

Comment:
Also should we have a separate PR for renaming `Tup` to `to_tuple`? Seems irrelevant to the rest of the changes. Not very important though.